src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/types.ts

```diff
@@ -56,10 +56,13 @@ export interface SuggestionRawDefinition {
   sortText?: string;
   /**
    * Suggestions can trigger a command by id. This is useful to trigger specific actions in some contexts
+   * https://microsoft.github.io/monaco-editor/typedoc/interfaces/languages.Command.html
    */
   command?: {
     title: string;
     id: string;
+    arguments?: any[];
```

Comment:
Glad these worked out! Consider opening an issue to fix the Monaco typings (https://github.com/microsoft/monaco-editor).